.github/workflows/maven.yml

```diff
@@ -1,4 +1,4 @@
-name: Maven Tycho build currently without the unit tests
+name: Maven Tycho build via Github actions
 
```

Comment:
Should we name it not by technology used but by actual goal? Like "Clean build"

Reply:
Thanks for this valuable contribution, I will adjust the name.